config/templates/archlinux.common.conf.in

```diff
@@ -30,3 +30,9 @@ lxc.stopsignal=SIGRTMIN+14
 # lxc.cap.drop = setpcap          # big big login delays in Fedora 20 systemd
 #
 lxc.cap.drop = setfcap sys_nice sys_pacct sys_rawio
+
+# fixes for systemd in lxc containers
+lxc.autodev = 1
+lxc.pts = 1024
+lxc.kmsg = 0
+lxc.hook.autodev = /var/lib/lxc/autodev
```

Comment:
You can't hardcode `/var/lib/lxc/autodev` here since it containers can be created somewhere else if a user chooses to do so. If you really want to include a hardcoded script to set up devices you should come up with a way that accounts for the fact that containers can be created anywhere on the system. It's also fine if you prefer to take out that section for now, get that patch merged and come back with a solution for this later.